search/double_linear_search.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <cassert>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```